.github/workflows/build.yml

```
@@ -0,0 +1,42 @@
+name: build
+on:
+  # Enable manually triggering this workflow via the API or web UI
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
+  pull_request:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v2
```

```suggestion
      - uses: actions/setup-go@v4
```
Why should we not use the latest version?

---

I forgot to bump it in that specific instance 👍🏻